Doppler.BigQueryMicroservice/Controllers/BigQueryController.cs

```diff
@@ -49,6 +65,28 @@ public async Task<ActionResult> SaveAllowedEmails(string accountName, AllowedEma
             var result = await _userAccessByUserRepository.MergeEmailsAsync(user.IdUser, model.Emails);
             return Ok(result);
         }
-    }
 
+        /// <summary>
+        /// Notify allowed users from doppler raly
+        /// </summary>
+        /// <param name="accountName">default account name from doppler</param>
+        /// <param name="emails">List of emails to notify</param>
+        /// <returns>true if not erros</returns>
+        [HttpPost("/big-query/{accountName}/notify-emails")]
+        public async Task<ActionResult> NotifyAllowedEmails(string accountName, EmailNotifier model)
```

Comment:
Does it mean that it is the responsibility of the API client to notify? 

In my opinion, this notification should be a side effect of `PUT /big-query/{accountName}/allowed-emails`, I mean, the API Client should not know about this notification.

<img width="974" alt="image" src="https://user-images.githubusercontent.com/1157864/134731843-b9ae2de4-7f29-4789-aa5c-6c36088054f9.png">

I am missing something?

Reply:
Solved en next push, the model definition was changed because the languaje is required for get the template information